src/bin/wasmer.rs

```diff
@@ -188,8 +192,10 @@ fn main() {
         CLIOptions::SelfUpdate => {
             println!("Self update is not supported on Windows. Use install instructions on the Wasmer homepage: https://wasmer.io");
         }
+        #[cfg(not(target_os = "windows"))]
         CLIOptions::Cache(cache) => match cache {
             Cache::Clean => {
+                use std::fs;
```

Comment:
Can we move this import to the head?

Reply:
@lachlansneff @bjfish do we mind this coding style? It is our preference. I moved it into this block because it is unused on the windows target.